### PR TITLE
bt-256:  selector animations

### LINKED ISSUE #256: Add slide animation to selector component
Currently, we have some jumpy animation on the selector list. Let's add some animations to it using react native reanimated library to improve UX a bit

### PR DESCRIPTION
**Pull request by mskmee**
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/85788646/c88290f8-90be-46b8-ab6d-9f7f867af85d

second version:

https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/85788646/50894093-d600-4e40-995d-0b67766fee3d

### CODE REVIEW

**Comment by katerynakostikova** [comment]
Let's fix this doubled bottom border 
image

**Comment by mskmee** [comment]
> Let's fix this doubled bottom border image

**Comment by mskmee** [comment]
> Let's fix this doubled bottom border image

image

File: mobile/src/bundles/common/components/selector/styles.ts [hunk 1]
```diff
@@ -2,18 +2,23 @@ import { StyleSheet } from 'react-native';
...
+    },
     dropdownButton: {
         borderWidth: 1,
         borderColor: Color.INPUT,
     },
     dropdown: {
         position: 'absolute',
         top: '95%',
-        maxHeight: 150,
+        maxHeight: SELECTOR_STYLE.MAX_DROPDOWN_HEIGHT,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/selector/styles.ts]: is this dropdown style used anywhere ?

Comment (mskmee) [reply mobile/src/bundles/common/components/selector/styles.ts]: Nope, it's unnecessary

**Review by katerynakostikova — APPROVED** [review]

**Review by anogire — APPROVED** [review]

**Pull request merged**